src/NServiceBus.Testing.Fakes/TestableAuditContext.cs

```diff
@@ -17,12 +18,12 @@ public class TestableAuditContext : TestableBehaviorContext, IAuditContext
         /// <summary>
         /// Address of the audit queue.
         /// </summary>
-        public string AuditAddress { get; set; }
+        public string AuditAddress { get; set; } = "audit queue address";
```

Comment:
I wanted the string to be as descriptive as possible. It's just some default value to avoid possible NullRef exceptions. I don't see anything wrong with the current value.